core/vibes/soul/sections/product-detail/index.tsx

```diff
@@ -8,154 +8,147 @@ import { ProductGallery } from '@/vibes/soul/sections/product-detail/product-gal
 import { ProductDetailForm, ProductDetailFormAction } from './product-detail-form';
 import { Field } from './schema';
 
-interface ProductDetailProduct {
```

Comment:
Flattened into `Props`. Will carry this over to the VIBES repo once this is approved.